src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForIndividualBuildingBlock.cs

```diff
@@ -23,6 +25,12 @@ public class InteractionTasksForIndividualBuildingBlock : InteractionTasksForPro
       {
       }
 
+      public IReadOnlyList<IndividualBuildingBlock> LoadFromPKML()
+      {
+         var filename = AskForPKMLFileToOpen();
+         return (string.IsNullOrEmpty(filename) ? Enumerable.Empty<IndividualBuildingBlock>() : LoadItems(filename)).ToList();
+      }
+
```

Comment:
Can this be pulled up to `InteractionTasksForPathAndValueEntity`. This code is the same as for `ExpressionProfileBuildingBlock` with the exception that the building block is different. That should be possible using the generic at the higher level class.

Reply:
Rob, I got your suggestion but...
Wouldn`t it be better to place it on "InteractionTaskFor" instead? since the suggested class "InteractionTasksForPathAndValueEntity" has this interface:
 IInteractionTasksForPathAndValueEntity<in TBuildingBlock, in TBuilder>
 
 Let me know!

Reply:
You can pull it up however high it makes sense. I think if you use my suggestion, you need to remove `in` from `TBuildingBlock`.

But, indeed, there is no reason it can't go up higher in the heirarchy.

Reply:
Great, thanks Rob!